src/dbus/server/dbus_thread_object.cpp

```diff
@@ -296,12 +328,14 @@ void DBusThreadObject::DetachHandler(DBusRequest &aRequest)
 
 void DBusThreadObject::FactoryResetHandler(DBusRequest &aRequest)
 {
+    otbrLogInfo("FactoryReset");
```

Comment:
Instead of add log for each command, can we log for all commands in a unified way? 
And seems we are already printing dbus log like `Handling method FactoryReset` ?

Reply:
It is hard for us to use a unified way. Because the parameters of each command are different, we need to print out all the parameters to get detailed info when the program crashed.
This log only record the Dbus API operations, other modules print out the `FactoryReset` log may not be caused by the Dbus. And I didn't find other modules have print the log `Handling method FactoryReset`.